gtk/Gir.toml

```
@@ -2646,6 +2646,9 @@ generate_builder = true
         [[object.function.parameter]]
         name = "iter"
         const = true
+    [[object.function]]
+    name = "new_with_attributes"
+    ignore = true # va_args
```

```suggestion
    ignore = true # va_args
```
don't use ignore, instead use manual = true & mark it as renamed as well.